products/views.py

```diff
@@ -1,3 +1,35 @@
-from django.shortcuts import render
+import json
 
-# Create your views here.
+from django.http      import JsonResponse
+from django.views     import View
+
+from products.models  import Product
+
+class ProductDetailView(View):
+    def get(self, request, product_id):
+        try:
+            product            = Product.objects.get(id = product_id)
+            product_images     = product.productimages.all()
+            information_images = product.informationimages.all()
+            product_sizes      = product.productsizes.all()
+            product_urls       = [image.url for image in product_images]
+            information_urls   = [information_image.url for information_image in information_images]
+            size_price         = [{'size' : product_size.size.size , 'price' : product_size.price} for product_size in product_sizes]
```

Comment:
다시 변수에 안 담고 아래와 같이 바로 사용해도 좋을 것 같네요.
```suggestion
        try:
            product            = Product.objects.get(id = product_id)
            product_urls       = [image.url for image in product.productimages.all()]
            information_urls   = [information_image.url for information_image in product.informationimages.all()]
            size_price         = [{
                'size' : product_size.size.size , 
                'price' : product_size.price
            } for product_size in product.productsizes.all()]
```

Reply:
넵 바로 컴프리헨션에서 진행했습니다!